### DISCUSSION TITLE
Welcome to roblox-clothing-scraper Discussions!

### DISCUSSION THREAD

**Discussion by AstreusCoding** [AstreusCoding/roblox-clothing-scraper#1] [Announcements]
## 👋 Welcome to the Clothing Scraper Community!
We're using Discussions as a place for anyone interested in our small project that scrapes clothing data from Roblox. Here's what we'd love from this community:

* **Ask questions** about using or extending the scraper.
* **Share your ideas** on how to improve or add features.
* **Engage with other members** who are curious or have suggestions.
* **Be supportive and open-minded** – this is a small but growing community that thrives on collaboration.

To get started, feel free to introduce yourself below and let us know how you're planning to use the scraper or if you have any cool ideas to add!

**Comment by dzemos** [comment]
How could I change the template for mine

**Reply by AstreusCoding** [reply]
This is actually a feature I'm working on at the moment. However I'm rewriting it all so it might be 20 - 30 hours before I push it to git.

**Reply by dzemos** [reply]
> This is actually a feature I'm working on at the moment. However I'm rewriting it all so it might be 20 - 30 hours before I push it to git.

Sure, thank you so much

**Comment by dzemos** [comment]
Also You can add auto uploader in the future
